.rubocop.yml

```diff
@@ -0,0 +1,7 @@
+AllCops:
+  Exclude:
+    - db/*.rb
+    - db/migrate/*.rb
```

Comment:
We could do an exception for `seeds.rb` but I think that's the only one of consequence.